release notes/v0.40.0.md

```diff
@@ -0,0 +1,253 @@
+k6 v0.40.0 is here! :tada: This release includes the promised experimental modules, some other small fixes and a lot of changes needed for the metric time series, unindexable tags and native ESM support. [Roadmap and future plans section](#roadmap-and-future-plans) makes another appearance with more info on how things are going and what should be expected
+
+# New Features and notable changes!
+
+## ⚠Breaking change⚠: Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+As part of the ESM changes, it was found that anything defined in the main module's scope was also accessible globally. This was because it was directly evaluated in the global scope.
+This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules, CommonJS or ESM, is to separate modules, this is obviously rather a bug than a feature.
+
+On the other note, we've seen [reports](https://github.com/grafana/k6/issues/2623) by people who have this global accessibility of the main module (intentionally or not). Still, it seems relatively rare, and only a few usages in a script. So if you need something accessed globally, our suggested workaround is to set it explicitly on the global object `globalThis`.
+
+## ⚠Breaking change⚠: `k6/ws` take `throw` option into consideration [#2247](https://github.com/grafana/k6/issues/2247)
+
+`k6/http` has been using the [`throw` option](https://k6.io/docs/using-k6/k6-options/reference/#throw) to figure out if should throw an exception on errors or whether it should return an response object with error set on it (and log it).
+
+This functionality is finally also available for `k6/ws` which previous to this would've *always* thrown an exception making scripting around it more involved. [#2616](https://github.com/grafana/k6/pull/2616)
+
+This also is a small breaking change as by default `throw` is `false` so it will now no longer throw an exception, but instead now it will return a Response with `error` property.
+
+Thank you @fatelei for making this change 🙇🎉!
```

Comment:
```suggestion
`k6/http` has been using the [`throw` option](https://k6.io/docs/using-k6/k6-options/reference/#throw) to figure out if it should throw an exception on errors or whether it should return a response object with an error set on it (and log it).

This functionality is finally also available for `k6/ws` which previous to this would've *always* thrown an exception making scripting around it more involved. [#2616](https://github.com/grafana/k6/pull/2616)

This also is a minor breaking change as, by default, `throw` is `false`, so it will now no longer throw an exception, but instead now, it will return a Response with `error` property.

Thank you, @fatelei, for making this change 🙇🎉!
```